app/src/main/java/com/example/pwmobilesdk/MainActivity.kt

```diff
@@ -58,6 +57,7 @@ class MainActivity : AppCompatActivity() {
         appBarConfiguration = AppBarConfiguration(navController.graph)
         setupActionBarWithNavController(navController, appBarConfiguration)
 
+        this._scrollViewTextLog      = findViewById(R.id.textview_log_scroll_view)
```

Comment:
I've now also checked different solutions and came to the conclusion, that the most simple solution would be to set this general behaviour on refering+initializing the `_scrollViewTextLog` in the following way:
```
Just add to your TextView:
android:gravity="bottom"
android:scrollbars = "vertical"

and set movement method:
myTextView.setMovementMethod(new ScrollingMovementMethod())
```
I've found this solution here:
https://stackoverflow.com/questions/8620153/how-to-get-the-android-textview-to-scroll-down-to-the-end-automatically

I think we can remove the explicit scroll method `scrollTextViewLog` in this way. :)